cmd/dvotecli/commands/file.go

```diff
@@ -0,0 +1,145 @@
+package commands
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"go.vocdoni.io/dvote/client"
+	"go.vocdoni.io/dvote/types"
+	"nhooyr.io/websocket"
+)
+
+var fileCmd = &cobra.Command{
+	Use:   "file",
+	Short: "file subcommands",
+}
+
+var fileAddCmd = &cobra.Command{
+	Use:   "add [name] [base64 payload]",
+	Short: "add file from a Base64 payload",
+	RunE:  fileAdd,
+}
+
+var fileFetchCmd = &cobra.Command{
+	Use:   "fetch [ipfs://<hash>]",
+	Short: "fetch file from URI, Base64 encoded",
+	RunE:  fileFetch,
+}
+
+var pinListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "list pinned files",
+	RunE:  pinList,
+}
+
+func init() {
+	rootCmd.AddCommand(fileCmd)
+	fileCmd.AddCommand(fileAddCmd)
+	fileCmd.AddCommand(fileFetchCmd)
+	fileCmd.AddCommand(pinListCmd)
+}
+
+func fileAdd(cmd *cobra.Command, args []string) error {
+
```

Comment:
Is this empty line allowed by "gofumpt" ? I think not. Are you using golanci-lint with the `.golangci.yml` file included in this repository?

Reply:
I installed all the required linters configuration you provided, and I didn't get any warning, but I will review.